index.html

```diff
@@ -0,0 +1,106 @@
+        <form action="" id="survey-form">
+          <div class="form__group">
+            <input type="text" class="form__group-input" id="name" placeholder="Name">
+            <label for="name" id="name-label" class="form__group-label">Name</label>
+          </div>
+  
+          <div class="form__group">
+            <input type="email" class="form__group-input" id="email" placeholder="Email">
+            <label for="email" id="email-label" class="form__group-label">Email</label>
+          </div>
+  
+  
+          <div class="form__group">
+            <input type="number" class="form__group-input" id="number" placeholder="Number" min="2" max="50">
+            <label for="number" id="number-label" class="form__group-label">Number</label>
+          </div>
+  
+          <div class="form__group">
+            <select class="form__group-select" name="dropdown" id="dropdown">
+              <option value="Male">Male</option>
+              <option value="Female">Female</option>
+              <option value="Other">Other</option>
+            </select>
+          </div>
+  
+          <div class="form-radio">
+            <div class="form__group-radio__button">
+              <input type="radio" class="form__group-input" id="radio-button" name="radio-button">
+              <label for="radio-button">Option 1</label>
+            </div>
+  
+            <div class="form__group-radio__button">
+              <input type="radio" class="form__group-input" id="radio-button" name="radio-button">
+              <label for="radio-button">Option 2</label>
+            </div>
+  
+            <div class="form__group-radio__button">
+              <input type="radio" class="form__group-input" id="radio-button" name="radio-button">
+              <label for="radio-button">Option 3</label>
+            </div>
+          </div>
+  
+          <div class="form-checkbox">
+            <div class="form__group-checkbox">
+              <input type="checkbox" class="form__group-input" value="dancing">
+              <label for="checkbox">Dancing</label>
+            </div>
+            <div class="form__group-checkbox">
+              <input type="checkbox" class="form__group-input" value="reading">
+              <label for="checkbox">Reading</label>
+            </div>
+          </div>
+  
+          <div class="form__group">
+            <textarea class="form__group-input" id="email"></textarea>
+            <label for="comments" class="form__group-label">Comments</label>
+          </div>
+  
+          <div>
+            <button type="submit" id="submit">Submit</button>
+          </div>
+  
+        </form>
+      </section>
+    </main>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
Please consider removing comments that contain unused code as it makes the code look untidy and less readable. This is the recommended practice :slightly_smiling_face: